baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -1139,3 +1113,67 @@ To configure Supplemental Data Storage per the policy:
 3.	Click the **Supplemental Data Storage** card.
 4.	Ensure the checkbox for "**Russian Federation**" is unchecked.
 6.	Click **Save**.
+
+
+## 16. Additional Google Services
+
+This section covers the oogle services that do not have an individual control and whether these serviGces are on or off.
+
+### Policy
+
+#### GWS.COMMONCONTROLS.16.1v0.2
+Service status for Google services that do not have an individual control SHOULD be set to OFF for everyone.
+
+- _Rationale_: Allowing access to additional google services without a need may create unnecessary vulnerabilities within the Google Workspace environment. By turning these services off, it mitigates the risk by not allowing access.
+- _Last modified:_ June 11, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - Pending
+
+### Resources
+-	[GWS Admin Help \| Turn on or off additional Google services](https://support.google.com/a/answer/181865)
+
+### Prerequisites
+
+- Super Admin role
+
+### Implementation
+
+#### GWS.COMMONCONTROLS.16.1v0.2 Instructions
+To configure additional services per the policy:
+1.	Sign in to the [Google Admin console](https://admin.google.com) as an administrator.
+2.	Navigate to **Apps** -> **Additional Google services**.
+3.	Click **CHANGE** at the top where it says if **Access to additional services without individual control for all organizational units is On/Off**.
+4.	Select the option: "**OFF for everyone**"
+6.	Click **Save**.
+
+## 17. Multi-Party Approval
+
+This section covers the oogle services that do not have an individual control and whether these serviGces are on or off.
+
+### Policy
+
+#### GWS.COMMONCONTROLS.17.1v0.2
+Require multi party approval for sensitive admin actions SHALL be enabled.
+
+- _Rationale_: Changes to certain admin settings creates the potential for security weaknesses. Therefore, requiring multiple super admins to approve changes to those settings mitigates the risk.
```

Comment:
"Therefore" implies that the second sentence will logically follow from the first, which it doesn't. Recommend re-working. Additionally, including an example of what's included in "certain admin settings" would make this rationale much more convincing.

```suggestion
- _Rationale_: Changes to sensitive admin settings, such as disabling 2-step verification, could introduce serious vulnerabilities in the GWS environment. Requiring multiple super admins to approve changes to those settings mitigates the risk changing these settings pose.
```